doc/source/examples/incompressible-flow/2d-taylor-couette-flow/2d-taylor-couette-flow.rst

```diff
@@ -0,0 +1,280 @@
+
+.. image:: images/lethe_analytical_taylor_couette_comparison.png
+    :alt: Azimuthal velocity compared with the analytical solution
+    :align: center
+
+
+The end of the simulation log provides the following information about the convergence of the error:
+
+.. code-block:: text
+
+  cells  error_velocity    error_pressure   
+    256 9.623524e-05    - 2.595531e-04    - 
+   1024 1.270925e-05 2.92 6.696872e-05 1.95 
+   4096 1.613718e-06 2.98 1.675237e-05 2.00 
+  16384 2.025381e-07 2.99 4.181523e-06 2.00 
+
+This table report the :math:`\mathcal{L}^2` norm of the error as a function of the number of cells. The third and the fifth column report the apparent order of convergence of the scheme. We see that the velocity converges at third order and the pressure at second order. This is exactly what is expected when using Q2-Q1 elements.
+
+.. note::
+  A curious reader will find that very similar results are obtained when using Q2-Q2 elements. For flows at low Reynolds number, using equal order elements for the pressure does not lead to a higher convergence rate. 
+  
+Finally, the simulation produces a file that contains the torque calculated on every boundaries. The file ``torque.00.dat`` contains the torque on ``bc 0`` and the file ``torque.01.dat`` contains the torque on ``bc 1``.
+
+For the boundary 0, the following torque are obtained:
+
+.. code-block:: text
+
+  cells     T_x          T_y           T_z      
+  256   0.0000000000 0.0000000000 -0.8192063151 
+  1024  0.0000000000 0.0000000000 -0.8319958810 
+  4096  0.0000000000 0.0000000000 -0.8361362739 
+  16384 0.0000000000 0.0000000000 -0.8373265692 
+
+
+For the boundary 1, the following torque are obtained:
+
+.. code-block:: text
+
+   cells     T_x          T_y          T_z      
+    256 0.0000000000 0.0000000000 0.8357077079 
+   1024 0.0000000000 0.0000000000 0.8372702342 
+   4096 0.0000000000 0.0000000000 0.8376393911 
+  16384 0.0000000000 0.0000000000 0.8377288180
+
+The analytical value of the torque is : :math:`T_z=0.837758`. Two main conclusions can be drawn. First, the torque obtained from the simulation on both boundaries converges to the analytical solution (at a second-order rate). Secondly, the torque on the difference between the torque on the outer and the inner cylinder converges to zero. This is what we would expect due to Newton's third law (action-reaction). However, it is only reached once the mesh is sufficiently fine and we note a significant (approx 2%) disagreement between the two torques for the coarsest mesh.
+
+Possibilities for extension
+----------------------------
+
+- Calculate formally the order of convergence for the torque :math:`T_z`.
+- It could ve very interesting to investigate this flow in 3D at a higher Reynolds number to see the apparition of the Taylor-Couette instability. This, however, would be a major undertaking. 
```

Comment:
```suggestion
- It could be very interesting to investigate this flow in 3D at a higher Reynolds number to see the apparition of the Taylor-Couette instability. This, however, would be a major undertaking. 
```